### ISSUE TITLE
Got an error "unknow option winbar"

### ISSUE THREAD

**Issue by thyeun** [crivotz/nv-ide#17]
Getting an error while launching nvim as below

![2022-06-26T23:03:12,140492376+08:00](https://user-images.githubusercontent.com/1939446/175820651-ef5979b6-7efe-4337-855b-9e1d694b8904.png)

**Comment by crivotz** [comment]
Hi, you will use the branch `neovim-stable` or move on neovim nightly.
The `winbar` has not yet been released in the `stable` version of Neovim

**Comment by thyeun** [comment]
@crivotz yup, i notice that, you have a discussion on this issue too. Arch linux nobody support/maintenance on ```neovim-nightly```, so have to manually install ```neovim-nightly```

**Comment by crivotz** [comment]
You can switch to neovim-stable branch.
It is perfectly aligned with the branch master, except for the winbar (you find the same information in the statusline as it was before) and for the management of autocmd (although they are now also managed in stable)

**Comment by thyeun** [comment]
@crivotz i'm on swaywm, not going to zsh, stay original with bash for the moment, scare will break my config on swaywm. but will sure try ```zinit``` (look delicious to handle github site) hahahaha.

**Comment by thyeun** [comment]
@crivotz my previous was i3wm with eww-bar (fancy :D ). But notice that i3wm memory consumption still abit higher than swaywm. Down side for swaywm, still many application not fully support wayland (especially Nvidia :( ). 

Come join the dark side (Swaywm) :))

**Comment by crivotz** [comment]
> @crivotz my previous was i3wm with eww-bar (fancy :D ). But notice that i3wm memory consumption still abit higher than swaywm. Down side for swaywm, still many application not fully support wayland (especially Nvidia :( ).
> 
> Come join the dark side (Swaywm) :))

I will start with `eww-bar`, I hadn't seen it yet. At the moment I'm still on polybar
